tools/sgen/input/FluentUIStyle.yml

```diff
@@ -286,6 +286,33 @@ MSFSubtleFloatingActionButtonTokens extends MSFButtonTokens:
   textFont: [ small: "Font(15px, regular)", medium: "Font(15px, regular)", large: "Font(17px, regular)" ]
   textColor: [ rest: $Colors.Foreground.neutral3, hover: $Colors.Foreground.neutral3, pressed: $Colors.Foreground.neutral3, selected: $Colors.Foreground.brandRest, disabled: $Colors.Foreground.neutralDisabled ]
 
+AP_MSFCardNudgeTokens:
+    accentColor: $Colors.Brand.shade20
+    accentIconSize: $Icon.size.xxSmall
+    accentPadding: $Spacing.xxSmall
+    backgroundColor: $Colors.Background.neutral2
+    buttonBackgroundColor: $Colors.Brand.tint30
+    buttonInnerPaddingHorizontal: $Spacing.small
+    buttonInnerPaddingVertical: $Spacing.xSmall
+    circleSize: $Icon.size.xxLarge
+    cornerRadius: $Border.radius.xLarge
+    iconSize: $Icon.size.xSmall
+    innerPadding: $Spacing.medium
```

Comment:
Since the innerPadding and outerHorizontalPadding are mapping to the same value, can we combine those into a single horizontalPadding token? (in the interest of saving space on the final binary size)

Reply:
Same for buttonInnerPaddingVertical and outerVerticalPadding.

Reply:
Good question. They're conceptually different properties - one describes the inner layout of the control, and the other describes how much of an external buffer to provide. It's just a coincidence that they map to the same value.

In these cases, is it best to combine now and split later if needed? (Not sure how we'd do that in a fully pipelined world.)

Reply:
As we quickly discussed offline, I think we can go with your strategy of "best to combine now and split later if needed" once the design team has all the formal token definitions. This way we can prevent adding more to the binary size right from the beginning.